static/bidder-info/insticator.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: "https://ex.ingage.tech/v1/prebidserver"
```

Comment:
endpoint is reachable. 

```
% curl -i --location --request POST 'https://ex.ingage.tech/v1/prebidserver'
HTTP/2 403
date: Tue, 06 Aug 2024 09:27:09
```


@shubhamc-ins are there any geo location restriction for insticator adapter? Because received following error as curl response

```

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Access Restricted</title>
    <style>
        body { font-family: Arial, sans-serif; text-align: center; padding: 50px; }
        h1 { color: #ff0000; }
    </style>
</head>
<body>
    <h1>Access Restricted</h1>
    <p>We're sorry, but access to this service is restricted from your location. Error code: 1</p>
    <p>If you believe this is an error, please contact our support team.</p>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'8aedfd4b2a2e3c5e',t:'MTcyMjkzNjQyOS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script></body>
</html>
```

Reply:
@onkarvhanumante yes, we do have geo restrictions on ingage.tech domain. You should be to access it via VPN using America/Canada region.